manifests/charts/global.yaml

```diff
@@ -454,6 +454,18 @@ global:
   # CSR clients such as the Istio Agent and ingress gateways can use this to specify the CA endpoint.
   caAddress: ""
 
+  # Config IstioAgent's CA Provider. Supported: Citadel, GoogleCA, KeyfactorCA
+  caProvider: "Citadel"
+  
+  # KeyfactorCA configurationsd
+  keyfactor:
+    secretName: ""
+    caName: ""
+    authToken: ""
+    appKey: ""
+    caTemplate: ""
+    metadata: []
```

Comment:
I think these have to go into the istio/api repo under the istio/api/mesh/v1alpha1/mesh.proto

Reply:
Thank @rshriram,
Those values are inputs from users and used for configuring Istio Installer.
I just follow the guidelines in manifests/charts/UPDATING-CHARTS.md

Reply:
The env variables can be overrided by k8s overlay setting in operator API, see: https://github.com/istio/istio/tree/master/operator#new-api-customization
An example:
```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  components:
    pilot:
      k8s:
        env:
        - name: KEYFACTOR_CA
           valueFrom:
             secretKeyRef:
               name: secretName
               key: "caName"
```
would add env `KEYFACTOR_CA` to pilot deployment.

Reply:
thank @morvencao for suggesting that, It seems a good point. 
But it requires advanced users to configure. It's could be better if users can explicitly configure through the YAML file and have a validator. 
An example:
![image](https://user-images.githubusercontent.com/49465776/83112512-75f7f100-a0f0-11ea-8f16-ef7f8d89fc53.png)

Reply:
I think we want to avoid introducing vendor-specific flags into global config if possible. What @morvencao suggests is the right way to go. Using ENV variable map is the way to integrate with Google CA. Keyfactor should be aligned as well. We can offer a good UG, and I don't think there will be any issue integrating with operator `--set` flags and even validator.